app/models/concerns/orchestration/dhcp.rb

```diff
@@ -79,7 +79,7 @@ def boot_server
     return unless tftp?
 
     # first try to ask our IPv4 TFTP server for its boot server
-    bs = tftp.bootServer
+    bs = subnet.tftp.setting('TFTP', 'tftp_servername') || tftp.bootServer
```

Comment:
So this reads setting and falls back to bootServer method. Is that fallback for compatibility reasons? Do we sill need it in the future? Maybe deprecating it using our helpers?